modules/openapi-generator/src/main/resources/rust-server/README.mustache

```diff
@@ -73,43 +73,23 @@ The examples can be run in HTTPS mode by passing in the flag `--https`, for exam
 cargo run --example server -- --https
 ```
 
-This will use the keys/certificates from the examples directory. Note that the server chain is signed with
-`CN=localhost`.
-
-
-## Writing a server
```

Comment:
What was wrong with this section?

Reply:
It's largely wrong and unhelpful.

Just copying and pasting the dependencies from the API crate is unhelpful at best.

Using a workspace isn't really a sensible idea - you should really just treat the generated crate as a dependency in it's own right - you shouldn't need to modify it.

The examples are demonstrations of how the crate can be used - they aren't templates for people to slavishly copy.